src/core/qgslegendsettings.h

```diff
@@ -272,57 +347,57 @@ class CORE_EXPORT QgsLegendSettings
      */
     void drawText( QPainter *p, const QRectF &rect, const QString &text, const QFont &font, Qt::AlignmentFlag halignment = Qt::AlignLeft, Qt::AlignmentFlag valignment = Qt::AlignTop, int flags = Qt::TextWordWrap ) const;
 
-    //! Returns a font where size is in pixel and font size is upscaled with FONT_WORKAROUND_SCALE
+//! Returns a font where size is in pixel and font size is upscaled with FONT_WORKAROUND_SCALE
```

Comment:
@nyalldawson , what's with the dox on-liners losing their intent?

Reply:
Not sure how that happened, fixed